frontend/packages/client/src/components/Community/JoinCommunityButton.js

```diff
@@ -84,19 +88,25 @@ export default function JoinCommunityButton({
   };
 
   const classNamesContainer = classnames(
-    'column is-narrow-tablet is-full-mobile is-align-self-center',
+    'column is-narrow-tablet is-full-mobile',
     {
       [classNames]: !!classNames,
+      [containerAlignment]: !!containerAlignment,
```

Comment:
since these are initialized by default they should never be undefined so you can just do 

```
const classNamesContainer = `column is-narrow-tablet is-full-mobile `${classNames} ${containerAligntment}`
```